modin/engines/base/io/text/text_file_dispatcher.py

```diff
@@ -26,9 +33,30 @@
 
 
 class TextFileDispatcher(FileDispatcher):
+    """
+    Class handles utils for reading text formats files.
+
+    Inherits some util functions for processing files from `FileDispatcher` class.
+    """
+
     @classmethod
     def get_path_or_buffer(cls, filepath_or_buffer):
-        """Given a buffer, try and extract the filepath from it so that we can
+        """
+        Extract path from `filepath_or_buffer`.
+
+        Parameters
+        ----------
+        filepath_or_buffer : str, path object or file-like object
+            `filepath_or_buffer` parameter of `read_csv` function.
+
+        Returns
+        -------
+        str or path object
+            verified `filepath_or_buffer` parameter.
+
+        Notes
+        -----
+        Given a buffer, try and extract the filepath from it so that we can
         use it without having to fall back to Pandas and share file objects between
```

Comment:
`Pandas` -> `pandas`

Reply:
Why do we capitalize Modin, Ray and other components but not Pandas?..

Reply:
Please, look at [this](https://github.com/modin-project/modin/issues/2934#issuecomment-820283431)

Reply:
Fixed.